pom.xml

```diff
@@ -170,6 +148,13 @@
             <version>${json4s.version}</version>
         </dependency>
 
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.12.7.1</version>
```

Comment:
should use  the version`2.13.4.2` . old versions have some vulnerabilities.

Reply:
good point! Upgraded it.